### PR TITLE
Closing CfP Index Update and twiRF #4

### PR DESCRIPTION
**Pull request by gnunicorn**
This PR replaces the call-for-papers-bit on the main page with a
call for mentors blob. Further more it contains edition #4 of
This week in Rust Fest blog post.

### CODE REVIEW

File: _posts/2016-06-29-this-week-in-rustfest-4-closing-cfp-call-for-mentors.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "This Week in RustFest 4: CfP closed, but we are still looking for mentors"
+socialImageSrc: '/assets/posts/twirf-4-mentors.png'
+---
+
+## CfP closed
+
+Last weeks announcement to extend the call for Papers was a big success. At closing time this Monday we had a total of <strong>30 valid</strong>  by <strong>29 awesome people</strong> 🎉🎊🎉, who want to present at the first RustFest in September. Unfortunately we don't have nearly enough time to have all talks presenting at the events, so the team will spend this week on the tough job of selecting talks and inform all speakers by July 7th.
+
```
Comment (badboy) [inline _posts/2016-06-29-this-week-in-rustfest-4-closing-cfp-call-for-mentors.md:10]: `at the event`, singular

**Comment by badboy** [comment]
Comment above, otherwise LGTM

**Comment by gnunicorn** [comment]
@badboy fixed. Merging then!

File: _posts/2016-06-29-this-week-in-rustfest-4-closing-cfp-call-for-mentors.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "This Week in RustFest 4: CfP closed, but we are still looking for mentors"
+socialImageSrc: '/assets/posts/twirf-4-mentors.png'
+---
+
+## CfP closed
+
+Last weeks announcement to extend the call for Papers was a big success. At closing time this Monday we had a total of <strong>30 valid</strong>  by <strong>29 awesome people</strong> 🎉🎊🎉, who want to present at the first RustFest in September. Unfortunately we don't have nearly enough time to have all talks presenting at the events, so the team will spend this week on the tough job of selecting talks and inform all speakers by July 7th.
```
Comment (skade) [inline _posts/2016-06-29-this-week-in-rustfest-4-closing-cfp-call-for-mentors.md]: I think this misses something. "30 valid _what?_".

Comment (gnunicorn) [inline _posts/2016-06-29-this-week-in-rustfest-4-closing-cfp-call-for-mentors.md]: yes. removed the wrong word. fixed it in gh-pages.

**Pull request merged**